kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
+
+# Enable server-side protection
+ssl_prefer_server_ciphers  on;
+ssl_ciphers                "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
```

Comment:
You should forbid null ciphers by adding `:!aNULL`. Maybe also forbid some very weak ciphers too.

Reply:
Done.